cmd/appendGlobalCmdFlagsToConfig.go

```diff
@@ -9,4 +9,5 @@ import (
 func AppendGlobalCmdFlagsToConfig(cmdFlags *pflag.FlagSet, config *lepton.Config) {
 	config.RunConfig.ShowWarnings, _ = cmdFlags.GetBool("show-warnings")
 	config.RunConfig.ShowErrors, _ = cmdFlags.GetBool("show-errors")
+	config.RunConfig.Debug, _ = cmdFlags.GetBool("debug")
```

Comment:
doesn't this conflict with existing '-d' / '--debug' flag? - maybe follow your convention of 'show-debug' -- or we could take the increasingly verbosity approach - we have '-v' - could have '-vv' and '-vvv' - or could change all of this to '-log-level=' ??

Reply:
There are no conflicts. The debug flag allows the interactive debugger and displays the debug messages. The only minor issue is that it overrides the flag description message.

But sounds good trying other flags approach. I was thinking of deprecating the `--show-errors` and `--show-warnings`. I like the `-v`,`-vv` and `-vvv` approach.
-v -> show errors
-vv -> show errors and information messages
-vvv -> show errors, warnings and information messages

Reply:
i'm ok w/that - just to be clear the debug flag should be a halt-the-world && launch the gdb listener - this is more for debugging api integration via debug log statements and these need to be 2 separate concerns, which is why I'm in favor of having a different naming convention of some kind, although this is kind of out of scope of this tkt so as long as there is no conflict (between the aforementioned) feel free to merge

Reply:
I'm adding the new flag `show-debug` to keep consistency with the names of the existing flags. We can review this on another ticket.